src/signalo/core/management/commands/init.py

```diff
@@ -17,8 +17,9 @@ def handle(self, *args, **options):
             "superuser": lambda: call_command("createsuperuser"),
         }
 
+        no_options = not any(options[k] for k in controller.keys())
```

Comment:
```suggestion
        no_options = not any(options.values())
```

Reply:
options contains more than the options you defined (defined by django, such as verbosity, etc.), so that won't work

Reply:
That sounds bad, because this opens the door to more bugs -- i.e. name collision between user-defined commands and default commands. We should probably define user commands in a subdict using a single key, then.

Reply:
There is no way to keep the user_defined commands isolated from Django's default commands?

Reply:
I've made a little push against this branch. Feel free to ask me to revert if you don't find it satisfactory.

Reply:
@olivierdalang Ok for you?

Reply:
sure